python/llm/example/GPU/LLM-Finetuning/patch_usage/finetune.py

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The only necessary code change to finetune with BigDL-LLM optimizations on Intel GPU
+from bigdl.llm import llm_patch
+llm_patch(train=True)
+
+# The following could be CUDA code of LLM finetuning application of PEFT
```

Comment:
# The following is the original LLM finetuning code using PEFT (without BigDL-LLM)